New apps addition

```diff
@@ -303,10 +320,12 @@ Table of Contents
 - [**Markor**](https://github.com/gsantner/markor) <sup>**[[F-Droid](https://f-droid.org/app/net.gsantner.markor)]**</sup>
 - [**Notally**](https://github.com/OmGodse/Notally) <sup>**[[F-Droid](https://f-droid.org/app/com.omgodse.notally)]**</sup>
 - [**neutriNote**](https://github.com/appml/neutrinote) <sup>**[[F-Droid](https://f-droid.org/app/com.appmindlab.nano)]**</sup>
+- [**Logseq**](https://github.com/logseq/logseq)
```

Comment:
From a quick overview it looks like this contains proprietary google components.

Reply:
Ok, removing it for now.

Request for F-Droid inclusion here: https://github.com/logseq/logseq/issues/4099
The developer seems interested, do give a thumbs up & subscriber issue thread for F-Droid related updates.

Reply:
:+1: already :-)

Reply:
Done

Reply:
Currently on [F-Droid](https://f-droid.org/packages/com.amaze.filemanager/) AmazeFileManager is too old.
The project is live & served on [izzy-repo](https://apt.izzysoft.de/fdroid/index/apk/com.amaze.filemanager).

IMO using old version is not good for privacy, shall we just remove the F-Droid link & keep source link?

Reply:
Sounds reasonable, yes!